lms/templates/discussion/_underscore_templates.html

```diff
@@ -440,7 +440,7 @@ <h1 class="home-title">${course.display_name_with_default}</h1>
 <script aria-hidden="true" type="text/template" id="thread-type-template">
     <div class="post-field">
         <div class="field-label">
-            <span class="field-label-text">
+            <span class="field-label-text" aria-describedby="field_help_post_type">
```

Comment:
So this aria-describedby should be on the fieldset below.  However, I noticed that we have a fieldset without a legend here (the legend is the span we're currently in, which means fieldset would have to be moved up).  The legend is actually the accessible label for the radio buttons and is a must.  If you don't want to use field set and legend here, an alternative would be something like this:
```
<div role=group aria-labelledby=post_type  aria-describedby=field_help_post_type>
  <p id=post_type">Post type:</p> 
  <input><label>
  <input><label>
</div>
```
Neither of these is a perfect solution.  Screen readers all handle the above differently.  Some will not read the description, but all should read the accessible label, so we should be good.